python/lsst/pipe/analysis/colorAnalysis.py

```diff
@@ -363,281 +564,1057 @@ def plotGalaxyColors(self, catalogs, filenamer, dataId):
-            np.median(distance[good]), 3.0*0.74*(q3 - q1), "{", "}", unitStr))
+            dataId, mean, stdDev, len(xx[keep]), len(xx), np.median(distance[good]),
+            3.0*0.74*(q3 - q1), "{", "}", unitStr))
     # The following is a Python 2 vs Python 3 issue
     try:
         log.info("Polynomial fit: {:2}".format(polyStr.translate(None, "${}")))
-    except:
+    except Exception:
         translationTable = dict.fromkeys(map(ord, "${}"), None)
         log.info("Polynomial fit: {:2}".format(polyStr.translate(translationTable)))
-    meanStr = "mean = {0:5.2f} ({1:s})".format(distance[good].mean(), unitStr)
-    stdStr = "  std = {0:5.2f} ({1:s})".format(distance[good].std(), unitStr)
+    meanStr = "mean = {0:5.2f}".format(mean)
+    stdStr = "  std = {0:5.2f}".format(stdDev)
+    rmsStr = "  rms = {0:5.2f}".format(rms)
+
+    count, bins, ignored = axes[1].hist(distance[good], bins=numBins, range=(-4.0*stdDev, 4.0*stdDev),
+                                        normed=True, color=polyColor, alpha=0.5)
+    axes[1].plot(bins, 1/(stdDev*np.sqrt(2*np.pi))*np.exp(-(bins-mean)**2/(2*stdDev**2)),
+                 color=polyColor)
+    axes[1].axvline(x=mean, color=polyColor, linestyle=":")
+    kwargs = dict(xycoords="axes fraction", ha="right", va="center", fontsize=7, color=polyColor)
+    axes[1].annotate(meanStr, xy=(0.34, 0.965), **kwargs)
+    axes[1].annotate(stdStr, xy=(0.34, 0.93), **kwargs)
+    axes[1].annotate(rmsStr, xy=(0.34, 0.895), **kwargs)
+
+    axes[1].axvline(x=0.0, color="black", linestyle="--")
     tractStr = "tract: {:d}".format(dataId["tract"])
-    axes[1].set_xlabel("Distance to polynomial fit (mag)")
-    axes[1].set_ylabel("Number")
-    axes[1].set_yscale("log", nonposy="clip")
-    axes[1].hist(distance[good], numBins, range=(-0.05*unitScale, 0.05*unitScale), normed=False)
-    axes[1].annotate(meanStr, xy=(0.6, 0.96), xycoords="axes fraction", ha="right", va="center",
-                     fontsize=8, color="black")
-    axes[1].annotate(stdStr, xy=(0.6, 0.92), xycoords="axes fraction", ha="right", va="center",
-                     fontsize=8, color="black")
     axes[1].annotate(tractStr, xy=(0.5, 1.04), xycoords="axes fraction", ha="center", va="center",
                      fontsize=10, color="green")
 
+    # Plot hardwired principal color distributions
+    if principalCol is not None:
+        pCmean = principalColor[kept].mean()
+        pCstdDev = principalColor[kept].std()
+        count, nBins, ignored = axes[1].hist(principalColor[kept], bins=bins, range=(-4.0*stdDev, 4.0*stdDev),
+                                             normed=True, color="blue", alpha=0.5)
+        axes[1].plot(bins, 1/(pCstdDev*np.sqrt(2*np.pi))*np.exp(-(bins-pCmean)**2/(2*pCstdDev**2)),
+                     color="blue")
+        axes[1].axvline(x=pCmean, color="blue", linestyle=":")
+        pCmeanStr = "{0:s}{1:s} = {2:5.2f}".format(wPerpStr[0:5], "$_{wired}$", pCmean)
+        pC
```

Comment:
Need 2 blank lines

Reply:
(I think this comment is in the wrong place...but I got it!)